sdk/src/types/api/core.rs

```diff
@@ -173,6 +174,31 @@ pub struct BaseTokenResponse {
     pub decimals: u32,
 }
 
+/// Information of a validator.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ValidatorResponse {
```

Comment:
`ValidatorInfoResponse` would be a kinda better name, similar to `NodeInfo` but :shrug:

Reply:
not important